### ISSUE TITLE
Sync without interactive prompt

### ISSUE THREAD

**Issue by sultanovich** [dooblem/bsync#12]
Hi dooblem,
I'm Sorry, this is not a issue but I don´t find other method to contact to you. 
I have one question, is it possible synchronised directories without interactive prompt?.
I want to use the cron daemon to automate the synchronization of a directory, but this will not be possible if I have an interactive prompt.

For example:

[root@aest-segu42 apeiti]# /root/bsync -v /home/apeiti/ apeiti@10.10.10.10:/home/apeiti

Password:

Loading filelists...
Loading original filelist from snap files...
Loading .bsync-snap-20140616160929.950125...
Loading dir1 filelist...
Loading dir2 filelist...
Analysing original paths...
Analysing remaining new paths in dir1...
Analysing remaining new paths in dir2...

(LEFT DIR CONTENT)                         (RIGHT DIR CONTENT)                   (ACTION)

```
                                  <--  A000552.dmp                           (copy)
```

passwd_acceso1                        <--  passwd_acceso1                        (sync)

Todo in /home/apeiti/: cp:1 sync:1
Todo in apeiti@10.10.10.10:/home/apeiti:
Apply actions? [y/N] y

Applying actions...
Applying actions in dir2...
Applying actions in dir1...
rsync: A000552.dmp
rsync: passwd_acceso1
Updating filelists...
Updating snap files: .bsync-snap-20140616161840.452684...
Done!
Cleaning SSH master...
[root@aest-segu42 apeiti]#

Thanks a lot

**Comment by Buridan** [comment]
Hi dooblem,
Why the PR for non-interactive mode isn't accepted yet?
If it's only a problem of parameter naming (-n or -b), could you simply accept it and make a revision after?
Thanks

**Comment by sultanovich** [comment]
Hi Burian,

If you can make the necessary changes I could test the new version in my UNIX lab. If we do this, dooblem only have to review and update the version.

Thanks

**Comment by dooblem** [comment]
Hi guys,
I intergrated scstarkey PR, and made the appropriate changes.
Can you test it ?
Thanks,
Marc

**Comment by Buridan** [comment]
Hi,
The Integration testing is working and -b parameter works! ... partially.
It fail as expected when there is conflict, but it doesn't say automatically Yes to copy & sync.
I don't know how it could be use in a cron job without this behavior;
perhaps a parameter like -y (--yes-to-all) is more relevant to describe this?
Thanks

**Comment by dooblem** [comment]
Sorry, I didn't see your PR before this message.
In my mind we should fix the -b (batch) mode.

**Comment by jgabriels** [comment]
I would also like this.  Ideally, it would autoaccept the left branch and create a <filename>.conflict<datestamp> file with the right content.   This should be relatively simple by making use of rsync's --backup and --suffix=  arguments.

**Comment by dooblem** [comment]
See also #23 . We may integrate this kind of solution.

.conflict files would be nice too.

**Comment by TB1234** [comment]
To fix the problem with the batch-mode change the following code:

Replace the code in line 1025 with the following code:
`if noninteractive:
     resp = "y"
else:
     resp = "non"`

This will make accept all changes automatically.

**Comment by dooblem** [comment]
thanks @TB1234 
fixed and pushed to master. Also added a test case for this bug.
Can you test it ?
note: the actual behaviour is still to exit on conflicts

**Comment by BurtGummer** [comment]
Little bit confused, is there a switch for that? There is no info about in the README.md?

**Comment by dooblem** [comment]
Yes, the README has to be updated. Just read the inline help of the command. There is a -b (batch) switch.
